Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityRaycastProvider.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Physics;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// The resulting hit information from an IMixedRealityRaycastProvider.
+    /// </summary>
+    public class MixedRealityRaycastHit
```

Comment:
This should likely be in it's own file

Reply:
Moved this out to a separate file in Definitions.